Giovaniavs/MyProject-server
# Hotfix(hotfix/mess-files): organazing the base files of the project

## What I did
* Organized some files in the root of the project.
## How to test
1. See if .github folder is on the root of the project.
2. See if there's any yarn files in the root of the project.
